TestProblem2.rb

```diff
@@ -0,0 +1,47 @@
+require_relative "Problem2"
+require "test/unit"
+ 
+class TestProblem2 < Test::Unit::TestCase
+ 
+  def test_correct_two_different_biggest_numbers
+
+    inputArray = Array(-4..9)
+
+    numbers= two_biggest_numbers(inputArray)
+   
+    assert_equal([9,8],numbers)
+    
+  end
+
+
+  def test_correct_two_equal_biggest_numbers
+
+    inputArray = [3,3,3,3,3,3]
+
+    numbers= two_biggest_numbers(inputArray)
+
+    assert_equal([3,3],numbers)
+  
+  end
+
+  def test_correct_with_one_number_array
+
+    inputArray = [5]
+
+    numbers= two_biggest_numbers(inputArray)
+
+    assert_equal([5],numbers)

```

Comment:
This differs from your initial implementation, you raised an error when the array was less than 2 in length.

Is the current approach better or you initial one, please justify your answer?